android/setup.cfg

```diff
@@ -51,14 +51,5 @@ zip_safe = False
 toga.backends =
     android = toga_android
 
-[options.extras_require]
-dev =
-    pre-commit
-    tox
-test =
-    pytest
-    pytest-tldr
-    coverage[toml]
```

Comment:
And I've removed the extras from all of the backend projects, because they all require toga-core anyway.